languages/swift/exercises/concept/enums/.docs/instructions.md

```diff
@@ -0,0 +1,61 @@
+In this exercise you'll be processing log-lines.
+
+Each log line is a string formatted as follows: `"[<LVL>]: <MESSAGE>"`.
+
+These are the different log levels:
+
+- `TRC` (trace)
+- `DBG` (debug)
+- `INF` (info)
+- `WRN` (warning)
+- `ERR` (error)
+- `FTL` (fatal)
+
+You have three tasks.
+
+## 1. Create LogLevel enum
+
+Define a `LogLevel` enum that has six cases corresponding to the above log levels, plus an unknown case for levels with missing or non-standard log levels.
+
+- `trace`
+- `debug`
+- `info`
+- `warning`
+- `error`
+- `fatal`
+- `unknown`
+
+Each case should have an associated String value that contains the log message.
```

Comment:
> Each case should have an associated String value that contains the log message.

I didn't completely understand what you meant by this.

Reply:
@wneumann See the above comment

Reply:
That was from an earlier draft of the exercise which used associated types instead of raw types and I forgot to remove that line. Thanks for catching it.